tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/IntegratedWindowsAuthUsernamePasswordTests.cs

```diff
@@ -154,7 +154,7 @@ internal void AddMockResponseForFederatedAccounts(MockHttpManager httpManager)
 
         private void AddMockResponseforManagedAccounts(MockHttpManager httpManager)
         {
-            httpManager.AddMockHandlerForTenantEndpointDiscovery(MsalTestConstants.AuthorityOrganizationsTenant);
+            httpManager.AddMockHandlerForTenantEndpointDiscovery(MsalTestConstants.AuthorityCommonTenant);
```

Comment:
This is also something that's concerning me a bit.  I've added URI validation across the board for these, and our code is sending the endpoint discovery request to Common in these cases.  So i'd like to understand this before I merge this PR.

---
In reply to: [248456275](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/791#discussion_r248456275) [](ancestors = 248456275)

Reply:
I thought it was because only work and school accounts were supported with these flows (IWA), and not personal accounts, so /common or /consumers will not work. can you try w/the desktop app and see what happens w/current msal v2 and compare w/v3? JM will know more.

---
In reply to: [248471515](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/791#discussion_r248471515) [](ancestors = 248471515,248456275)

Reply:
DefaultAuthority (which these were getting setup with) is /common so it was sending common here.  I've updated the tests to AddKnownAadAuthority(AadAuthorityAudience.AzureAdMultipleOrgs, true) so that we hit /organizations and this now looks/behaves properly.

---
In reply to: [248473112](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/791#discussion_r248473112) [](ancestors = 248473112,248471515,248456275)